packages/playwright-test/src/cli.ts

```diff
@@ -175,7 +169,9 @@ async function runTests(args: string[], opts: { [key: string]: any }) {
   config._internal.passWithNoTests = !!opts.passWithNoTests;
 
   const runner = new Runner(config);
-  const status = await runner.runAllTests();
+  if (opts.watch)
+    process.stdout.write('\x1Bc');
```

Comment:
```suggestion
    process.stdout.write('\x1Bc'); // Reset the terminal
```

Not sure if its worth mentioning with the other commands.